libmamba/include/mamba/specs/match_spec.hpp

```diff
@@ -136,22 +136,26 @@ namespace mamba::specs
          */
         [[nodiscard]] auto contains_except_channel(const PackageInfo& pkg) const -> bool;
 
-        auto operator==(const MatchSpec& other) const -> bool
+        // TODO: only use the `= default` implementation of `operator==` when we will use C++20.
+        [[nodiscard]] auto operator==(const MatchSpec& other) const -> bool
         {
-            return m_channel == other.m_channel && m_version == other.m_version
-                   && m_name == other.m_name && m_build_string == other.m_build_string
-                   && m_name_space == other.m_name_space && m_build_number == other.m_build_number
+            return m_channel == other.m_channel               //
```

Comment:
remove empty comments, same for others

Reply:
This allows formatting with one pair of attribute per line. Are there other ways to format such lines?

Reply:
Is this because the automatic formatting forces to do that? Otherwise I dont get why the `//` are necessary at all?

Reply:
If it's because of the automatic formatting, feel free to ignore my remark.